doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster configuration.
+Instructions for configuring TPUs on GKE can be found here:
+
+- {ref}`kuberay-gke-tpu-cluster-setup`
+
+## About TPUs
+
+TPUs are custom-designed AI accelerators which are optimized for training and inference of large AI models. A TPU host is a VM that runs on a physical computer connected to TPU hardware. TPU workloads can run on one or multiple hosts. A TPU Pod slice is a collection of chips all physically colocated and connected by high-speed inter chip interconnects (ICI). Single-host TPU Pod slices contain independent TPU VM hosts and communicate over the Data Center Network (DCN) rather than ICI interconnects. Multi-host TPU Pod slices contain two more interconnected TPU VM hosts. In GKE, multi-host TPU Pod slices run on their own node pools and are scaled atomically by node pools, rather than individual nodes. Both single-host and multi-host TPU Pod slices can be scaled seamlessly to multiple slices, enabling greater parallelism to support larger workloads.
+
+## Quickstart: Serve a Stable Diffusion model on GKE with TPUs
+
+After setting up a GKE cluster with TPUs and the Ray TPU initialization webhook, you're ready to begin running workloads on Ray with TPUs. The :ref:`StableDiffusion example <kuberay-tpu-stable-diffusion-example>` shows how to serve a model with Ray on single-host TPUs.
```

Comment:
![image](https://github.com/user-attachments/assets/74ebdf63-8738-4f1f-ae8b-788fa3a70d10)

This doesn't link correctly.

Reply:
Changed the link to a valid .md reference in [2a6a93f](https://github.com/ray-project/ray/pull/45924/commits/2a6a93fef8436a802b85f334d779f1b54c4d2d6a).